tabs/src/components/code/bot/adaptiveCard/SDK/TeamsBotSsoPromptCode.txt

```diff
@@ -0,0 +1,12 @@
+// @microsoft/teams-fx
```

Comment:
For js code snippet, you can use .js as extension name, so that in the editor it will have highlight and intellisense

Reply:
ok, I will modify it asap.